articles/140_topic_and_service_name_mapping.md

```diff
@@ -265,7 +335,7 @@ Here are some examples of translations between ROS topic names and the correspon
 | `/_foo/_bar`      | Topic     | `rt___foo___bar`       |
 | `/_foo/_bar/_baz` | Topic     | `rt___foo___bar___baz` |
 
-### ROS Topic and Service Name Length Limit
+#### ROS Topic and Service Name Length Limit
```

Comment:
This section still applies and shouldn't be under the "alternatives" section. It has to be re-worked to account for both the limit of partition length and topic name length